src/routes/products.ts

```diff
@@ -0,0 +1,78 @@
+import express, { NextFunction, Request, Response } from 'express';
+import { createProduct, deleteProduct, getProducts, updateProduct } from '../controllers/products';
```

Comment:
Let's also create an index file in the controllers folder. After that this import should change to 
```suggestion
import { createProduct, deleteProduct, getProducts, updateProduct } from '../controllers';
```